public/_redirects

```diff
@@ -209,3 +209,12 @@ https://docs.cert-manager.io/* https://cert-manager.io/docs/:splat 302!
 /docs/installation/upgrading/* /docs/releases/upgrading/:splat 301!
 /docs/release-notes/* /docs/releases/release-notes/:splat 301!
 /docs/installation/supported-releases/ /docs/releases/ 301!
+
+# Installation section cleanup
+/docs/installation/other-tools/ /docs/installation/helm/ 301!
+/docs/installation/verify/ /docs/installation/kubectl/ 301!
+
+# Moved the "upgrading" and "release-notes" pages to the release section
+/docs/installation/upgrading/* /docs/releases/upgrading/:splat 301!
+/docs/release-notes/* /docs/releases/release-notes/:splat 301!
+/docs/installation/supported-releases/ /docs/releases/ 301!
```

Comment:
I noticed some missing redirects. See comments above.
Please check that all deleted pages and all moved pages have corresponding redirects.